src/services/tasks.js

```diff
@@ -104,8 +104,14 @@ export const deleteTask = async (id, dispatch) => {
 export const toggleTaskCompletion = async (id, dispatch) => {
   const config = getConfig();
 
+  console.log(config);
```

Comment:
Suggestion: Can remove console calls in production unless important for future development

Reply:
thanks for pointing out, will delete it